app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml

```diff
@@ -10,7 +10,7 @@
     define('analyticsPopupConfig', function () {
         return {
             analyticsVisible: <?= $block->getNotification()->isAnalyticsVisible() ? 1 : 0; ?>,
-            releaseVisible: <?= $block->getNotification()->isReleaseVisible() ? 1 : 0; ?>,
```

Comment:
This change is not needed if it works with trailing comma.

Reply:
I agree with @orlangur here, keeping comma's in javascript objects or php arrays is a good thing, it gives a clearer diff in git when one or multiple lines are added if a comma already existed on the line above the new line.

For javascript, historically this was an issue in older Internet Explorer versions, they couldn't handle that, but according to [this comment](https://stackoverflow.com/a/19791620/145829) it was fixed in IE9 and higher and Magento only supports IE11, so we should be fine.

PHP is more and more encouraging this kind of syntax btw, PHP 7.3 got a new feature where arguments in function/method calls can now also have a trailing comma: https://www.php.net/manual/en/migration73.new-features.php#migration73.new-features.core.trailing-commas

Reply:
@orlangur, comma added after array, 
Thanks for suggestion.